app/services/analytics/ecf_school_service.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module Analytics
+  class ECFSchoolService
+    class << self
+      def update_school_analytics
+        return unless %w[test development production].include? Rails.env
```

Comment:
What's the motivation for this line?

Reply:
Saving money? 😂 

Do we have analytics database outside of those environments?

Reply:
yeah, those are the only environments the database exists

Reply:
And yes, the motivation for that is saving £100 per month per environment 😛

Reply:
💰 💰 💰